website/src/components/attributions/creative-commons.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import Link from "../link"
+import * as css from "./creative-commons.css"
+
+interface Props {
+    title: string,
+    authors: Array<{name: string, link?: string}>
+  }
+
+export const CreativeCommonsBy = (props: Props) => {
+    const licenceAttributes = {
+        'xmlns:cc':"http://creativecommons.org/ns#",
+        'xmlns:dct':"http://purl.org/dc/terms/"
+    };
+    const authors = props.authors.length-1;
```

Comment:
my one suggestion would be to maybe name this constant something else like authors_count? it tripped me up a bit at first since the array is also named authors in the props. super small change tho!

Reply:
Incorporated this in a new commit!